projects/storefrontlib/src/cms-components/myaccount/my-coupons/my-coupons.component.service.spec.ts

```diff
@@ -64,7 +68,11 @@ describe('MyCouponsComponentService', () => {
           query: RELEVANCE + CUSTOMER_COUPON_CODE + 'CouponForPartProduct',
         },
       },
-      { couponcode: 'CouponForPartProduct' }
+      {
+        state: {
+          couponcode: 'CouponForPartProduct',
+        },
+      }
```

Comment:
we should use property `queryParams` for query params, but not `state`; similar to https://github.com/SAP/spartacus/pull/12638#discussion_r648196574